README.md

```diff
@@ -54,6 +54,32 @@ Starting from zone.js v0.8.9, you can choose which web API module you want to pa
 For more details, please
 see [MODULE.md](MODULE.md).
 
+## Bundles
+There are several bundles under `dist` folder.
+- `zone.js`, the default bundle, contains the most used APIs such as `setTimeout/Promise/EventTarget...`, also this bundle will support all evergreen and legacy (IE/Legacy Firefox/Legacy Safari) Browsers
+- `zone-evergreen.js`, the bundle for evergreen browsers, will not include the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`.
+- `zone-legacy.js`, the patch bundle for legacy browsers, will only include the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`. Need to work with `zone-evergreen.js`.
```

Comment:
Do you mean: "This bundle must be loaded after zone-evergreen.js."?

Reply:
Yes, I will update this.